tests/acceptance/features/webUIFilesList/fileList.feature

```diff
@@ -12,11 +12,11 @@ Feature: User can view files inside a folder
     And user "Alice" has uploaded file "data.zip" to "data.zip" in the server
     And user "Alice" has logged in using the webUI
 
-  Scenario: Resources are listed
+  Scenario: Resources are listed and files list displays quicklink quick action
     When the user browses to the files page
+    Then quick action "quicklink" should be displayed on the webUI
```

Comment:
@kulmann @individual-it I merged the scenario from (formerly) line 28 here because running through the whole setup for a single assertion seemed overblown:)